devicetypes/ezex/somfy-blind-controller-ezex.src/somfy-blind-controller-ezex.groovy

```diff
@@ -0,0 +1,123 @@
+/**
+ *
+ *	Copyright 2019 eZEX
```

Comment:
Correct me if I'm wrong but I believe this DTH is developed by SmartThings and not eZEX so I think it should say SmartThings here and also in the `definition`

Reply:
It was a little unclear to me. At the first time, eZEX gave me the DTH that was partially working but contained some wrong implementation. I think it could changed to SmartThings. But do we need to get an agreement on it from the manufacturer? What is the standard normally?

Reply:
@lukhong It would depend on where they got the DTH from. I suspect, given the coding style, from the SmartThingsPublic project.

What might be appropriate (and actually technically required, I believe, but I am not a lawyer), if eZEX started with a ST DTH and added changes, would be:

```
Copyright 2019 SmartThings
Copyright 2019 eZEX
```

Regardless (and, disclaimer again, I am not a lawyer, so take the following as my understanding of how this stuff works), if they started with a ST DTH the `Copyright 2019 SmartThings` part (or, whatever the copyright date is in the DTH they used) is mandatory.

Reply:
Then I think it could be changed to SmartThings but I am not clear yet.
How was the [SDS doorlock](https://github.com/SmartThingsCommunity/SmartThingsPublic/tree/master/devicetypes/samsungsds/samsung-smart-doorlock.src) case before? It might be started from SmartThings DTH and same style.

Reply:
I just changed related things to SmartThings.